Customizing name of state folder - Does not automatically convert folder from camel case to kebab case

Using the following to create a customizable state folder: 
`ng g ngrx myState --directory=myState --module=libs/myFeature/myFeature.module.ts`

It will create a folder called myState, still camel-cased. However, the file will be changed over to kebab case as all angular cli commands do. In addition, the files themselves, will expect the folder to be kebab cased. 
```
Cannot find module './my-feature/my-feature.reducer'.
Cannot find module './my-feature/my-feature.init'.
Cannot find module './my-feature/my-feature.effects'.
```

--- a/packages/schematics/src/collection/ngrx/index.ts
+++ b/packages/schematics/src/collection/ngrx/index.ts
@@ -134,8 +134,9 @@ function addNgRxToPackageJson() {
   };
 }
 
-export default function(options: Schema): Rule {
+export default function(schema: Schema): Rule {
   return wrapIntoFormat(() => {
+    const options = normalizeOptions(schema);
     const name = options.name;
     const moduleDir = path.dirname(options.module);
 
@@ -150,4 +151,8 @@ export default function(options: Schema): Rule {
       ]);
     }
   });
+}
+
+function normalizeOptions(options: Schema): Schema {
+  return { ...options, directory: toFileName(options.directory)};
 }

--- a/packages/schematics/src/collection/ngrx/ngrx.spec.ts
+++ b/packages/schematics/src/collection/ngrx/ngrx.spec.ts
@@ -93,7 +93,7 @@ describe('ngrx', () => {
     expect(appModule).toContain('EffectsModule.forFeature');
     expect(appModule).not.toContain('!environment.production ? [storeFreeze] : []');
 
-    expect(tree.exists(`/apps/myapp/src/app/myCustomState/state.actions.ts`)).toBeTruthy();
+    expect(tree.exists(`/apps/myapp/src/app/my-custom-state/state.actions.ts`)).toBeTruthy();
   });
 
   it('should only add files', () => {

